app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_like_count
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Good job implementing the "update_counter" method. However, we can make this better by using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) methods `after_save` to call the method every time a new `like` has been saved to the database, then make this method private.